AEPUserProfile.podspec

```diff
@@ -10,10 +10,10 @@ Pod::Spec.new do |s|
   s.author           = "Adobe Experience Platform SDK Team"
   s.source           = { :git => "https://github.com/adobe/aepsdk-userprofile-ios", :tag => s.version.to_s }
 
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '12.0'
   s.swift_version    = '5.1'
```

Comment:
Should bump up the version here?

Reply:
@swarna04 The swift version is remaining the same.